packages/debian/entrypoint.sh

```diff
@@ -36,10 +36,27 @@ rpm2deb() {
 }
 
 
+buildrepo() {
+    set -x
+    mkdir -p /var/www/debian/{conf,incoming}
+    # We assume that all downloaded packages are mounted in /metalk8s-deb/
+    cp /metalk8s-deb/distributions /var/www/debian/conf
+    # Mount `distributions` file in  /var/www/debian/conf
+    reprepro -b /var/www/debian export
+    for dir in /metalk8s-deb/metalk8s-*; do
```

Comment:
So, IIUC (this loop would benefit from a comment explaining what it does, for readers unaware of Debian repositories building practices), we're adding all packages for `metalk8s-*` components.

Don't we also need to do the same for the `main` component? At least that's what I would expect from the `distributions` file.

Reply:
In fact there is no main component, I will remove it from the `distributions` file.